src/utilities/bcl/BCLMeasure.cpp

```diff
@@ -59,7 +59,7 @@ static constexpr std::array<std::pair<std::string_view, std::string_view>, 3> ap
   {"resources", "resource"},
 }};
 
-static constexpr std::array<std::string_view, 1> ignoredSubFolders{"tests/output"};
+static constexpr std::array<std::string_view, 3> ignoredSubFolders{"tests/output", "__pycache__", "tests/__pycache__"};
```

Comment:
When updating BCLMeasure / copying it, do not grab `__pycache__`

At some point we may want to also not grab anything .gitignored